### DISCUSSION TITLE
Preference for type for multiple argument names?

### DISCUSSION THREAD

**Discussion by mikeweltevrede** [pytest-dev/pytest#12591] [Q&A] [answered]
Hi all, curious as to your opinion on the following.

We have started to use [Ruff](https://github.com/astral-sh/ruff) for linting. In Ruff, the default for the expected type for multiple argument names is a `tuple`: https://docs.astral.sh/ruff/settings/#lint_flake8-pytest-style_parametrize-names-type. Example:
```python
@pytest.mark.parametrize(("name1", "name2"), ...)
```

In the [Pytest docs](https://docs.pytest.org/en/7.1.x/how-to/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions), I see everything is using the `"csv"` style. Example:
```python
@pytest.mark.parametrize("name1,name2", ...)
```

My personal tendency would be to prefer a list or tuple as well for clearer separation between the argument names.

My questions:
- Do others in the community have any preference?
- What are the reasons to use `"csv"` over `"tuple"` or `"list"`?

**Comment by RonnyPfannschmidt** [comment, marked as answer]
the commonly used style is csv, as it simply removes code noise

the flake8-pytest-style plugin that was used as the base for the ruff rules is using some defaults that do not match the pytest upstream recommendations

they recently started to make some concessions (like flipping the default for enforcing parens on marks)

**Reply by mikeweltevrede** [reply]
Thanks @RonnyPfannschmidt, clear answer!
